### ISSUE TITLE
Poinst Against store the fantasy points instead of the game points

### ISSUE THREAD

**Issue by rmayf** [rmayf/CollegeFootballChallenge#29]
It is confusing when you are looking at the stats...

**Comment by keylime503** [comment]
In the DB I assume? Not just when rendering?

**Comment by rmayf** [comment]
Yeah that's kinda what I was thinking.  I just don't want to hardcode the fantasy point per points against levels into the view.  We should only have that logic in one place in case it changes. Otherwise we can have another field in the table to store this
